[jit] delete netdef converter

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #34515 [jit] kill script namespace
* **#33807 [jit] delete netdef converter**

afaik this is unused, so removing it from the source tree. RIP :(

Differential Revision: [D20122118](https://our.internmc.facebook.com/intern/diff/D20122118)

suo committed:
[jit] delete netdef converter

afaik this is unused, so removing it from the source tree. RIP :(

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Looks good if it's not used!

suo committed:
Update on "[jit] delete netdef converter"


afaik this is unused, so removing it from the source tree. RIP :(

Differential Revision: [D20122118](https://our.internmc.facebook.com/intern/diff/D20122118)

[ghstack-poisoned]

suo committed:
Update on "[jit] delete netdef converter"


afaik this is unused, so removing it from the source tree. RIP :(

Differential Revision: [D20122118](https://our.internmc.facebook.com/intern/diff/D20122118)

[ghstack-poisoned]

suo committed:
Update on "[jit] delete netdef converter"


afaik this is unused, so removing it from the source tree. RIP :(

Differential Revision: [D20122118](https://our.internmc.facebook.com/intern/diff/D20122118)

[ghstack-poisoned]
